File: testdata/clojure/sample.clj
(ns sample.core
  (:require [clojure.string :as str]
            [clojure.set :as set])
  (:import (java.util Date)))

(def version "1.0.0")

(defn greet
  "Greets a user."
  [name]
  (str "Hello, " name))

(defn- internal-helper
  [x]
  (* x 2))

(defn process-items
  [items]
  (map str/upper-case items))

File: testdata/fsharp/sample.fs
module Sample

open System
open System.IO
open System.Collections.Generic

let greet name =
    sprintf "Hello, %s" name

let add x y = x + y

let private helper msg =
    printfn "%s" msg

member this.Process(items) =
    items |> List.map (fun x -> x * 2)
